### DISCUSSION TITLE
Add Artisan command to purge all ray statements

### DISCUSSION THREAD

**Discussion by rostockahoi** [spatie/laravel-ray#143] [Ideas]
Hey, wouldn't it be neat to have a command to remove all ray statements across the codebase? It could look like this when running Artisan:
```
$php artisan
// ...
ray
  ray:publish-config            Create the Laravel Ray config file in project root.
  ray:purge                     Remove all Laravel Ray statements
```

**Comment by freekmurze** [comment]
It would be very nice, but it's next to impossible to make it work well in all circumstances.

Feel free to PR your attempt.

**Comment by rostockahoi** [comment]
My first idea would be some Regex replace magic but Regex? Brrrr 

Von meinem iPhone gesendet

**Reply by sinnbeck** [reply]
https://github.com/spatie/ray/discussions/869
